pywbem_mock/_mainprovider.py

```diff
@@ -1889,10 +1889,11 @@ def _get_reference_instnames(self, namespace, instname, result_class,
         instname.namespace = namespace
         role = role.lower() if role else None
 
-        # Future TODO/ks: Search very wide here.  Not isolating to associations
-        # and not limiting to expected result_classes. Consider making list from
-        # get_reference_classnames if classes exist, otherwise set list to
-        # instance_store to search all instances
+        # ISSUE 2662 Future TODO/ks: Search very wide here.  Not isolating to
```

Comment:
Since we now have an issue I think we should remove the "TODO" word so it no longer shows up in the search for remaining TODOs. That's how it was done in some other files, e.g. in pywbem/_mof_compiler.py or pywbem/_tupleparse.py. The syntax used there was:
```
# Issue #1234: bla
```

Reply:
DONE